symbolic-ppdb/src/format/mod.rs

```diff
@@ -359,3 +367,85 @@ pub struct Document {
     pub name: String,
     pub(crate) lang: Language,
 }
+
+/// An iterator over Embedded Sources.
+#[derive(Debug, Clone)]
+pub struct EmbeddedSourceIterator<'object, 'data> {
+    ppdb: &'object PortablePdb<'data>,
+    inner_it: CustomDebugInformationIterator<'data>,
+}
+
+impl<'object, 'data> EmbeddedSourceIterator<'object, 'data> {
+    fn new(ppdb: &'object PortablePdb<'data>) -> Result<Self, FormatError> {
+        // https://github.com/dotnet/runtime/blob/main/docs/design/specs/PortablePdb-Metadata.md#embedded-source-c-and-vb-compilers
+        let embedded_sources_kind = Uuid::parse_str("0E8A571B-6926-466E-B4AD-8AB04611F5FE")
+            .map_err(|e| FormatError::new(FormatErrorKind::InvalidStringData, e))?;
```

Comment:
This can be done statically:
```suggestion
        const EMBEDDED_SOURCES_KIND: Uuid = uuid::uuid!("0E8A571B-6926-466E-B4AD-8AB04611F5FE");
```